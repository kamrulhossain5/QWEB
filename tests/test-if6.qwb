function int cond(bool b){
  int x;
  x = 10;
  if (b)
    if (x == 10)
      x = 19;
  otherwise
    x = 17;
  output x;
}

function int main(){
 print(cond(true));
 print(cond(false));
}
