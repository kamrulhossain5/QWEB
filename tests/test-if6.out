19<br>
10<br>
